### ISSUE TITLE
WSA is the best thing that has ever happened to windows.

### ISSUE THREAD

**Issue by Dalmacc** [microsoft/WSA#547]
### Is your feature request related to a problem? Please describe

...

### Describe the solution you'd like

...

### Describe alternatives you've considered

WSA is the flagship feature of Windows 11. By adding functionalities like that and improving the ecosystem, you can at least compete with MacOS. Almost everything in the Windows system is implemented terribly, but WSA is one of those things that works well, and Microsoft wants to destroy it and roll back Windows 11 to the reputation of Vista.

### Please specify the version of Windows Subsystem for Android

whyyy

**Comment by lisyclover** [comment]
I think so

**Comment by Slion** [comment]
WSA is indeed working really well and most useful as it is. Hopefully they will pick it up again.

**Comment by AgrMayank** [comment]
True, it is one of the best things that has ever happened to Windows!!

**Comment by lyc280705** [comment]
I think WSA is important to Windows on arm,considering the performance of Snapdragon X Elite.

**Comment by hifihedgehog** [comment]
It is absolute foolishness and shortsightedness from Satya Nadella and his cohorts in the executive chair to ax WSA.

Let me spell it out as Satya has been lacking in the logic department lately with the blunder of Copilot+ to answer for. 

Windows 11 lacks a mobile app library in the Microsoft Store. Microsoft excels in the triple E--embrace, extend, extinguish.

You are doing the polar opposite by axing this. Really.

Apparently, 2nd generation Snapdragon X next year brings further efficiency improvements to enable smaller and more passive designs. With it, you are going to release an 11" Surface Pro, and competitors will likely be bringing smaller tablets as well... without a competent app library to entice users.

Software trumps hardware always. Look at NVIDIA. Look at Tesla.

No one buys a tablet, especially smaller tablets where touch comes first and foremost in regular usage, without tablet-optimized, touch-friendly apps.

The beauty of WSA is it enables users to have tablet-optimized, touch-friendly apps from Android despite Microsoft Store's shortcomings, its admission of failure with ~~Progressive Web Apps~~  glorified web bookmarks lately replacing most real modern apps.

You could be literally embracing and extending Android to stay ahead while having a super power Android could never dream of: *bringing* full Windows program support along with access to Android's 2-million-plus touch-optimized apps.

That's win-win, but you'd rather have loss-lose due to myopia.

DeX and Stage Manager can never dream of this cornucopia of app compatibility and you throw it away, despite all of these software engineers' painstaking efforts.

Never mind you stupidly got rid of Windows 10's Tablet Mode, which us users can thankfully still bring back with https://github.com/valinet/ExplorerPatcher.

Never mind you can justify keeping Windows Subsystem for Linux without it bringing a revenue flow. Axing WSA because the Amazon Appstore partnership failed is throwing the baby out with the bathwater.

You are staring at your very advantage over Android, macOS, iPadOS, and iOS right in the face, and casting it aside as if it were nothing.

Don't be stupid [as you have been with countless graveyard casualties.](https://microsoftgraveyard.com/) WSA is the way forward to stay ahead and is your upper hand to keep an ace up your sleeve. Don't be stupid for once, please.

**Comment by AgrMayank** [comment]
@hifihedgehog well said! WSA could've been one of the core pillars of Windows especially with the new ARM chips. It's mind-blowing and emulators don't even come close. The fact that you cannot easily differentiate between a native and a WSA app says how good it was in terms of support and everything. 

After shutting it down they are planning to release it in China only with Tencent partnership. If Amazon has failed why not just have something like APKMirror to get the apps or leave the storefront to us. This is beyond stupid and another massive mistake after Lumia.

**Comment by lyc280705** [comment]
> @hifihedgehog well said! WSA could've been one of the core pillars of Windows especially with the new ARM chips. It's mind-blowing and emulators don't even come close. The fact that you cannot easily differentiate between a native and a WSA app says how good it was in terms of support and everything. 
> 
> After shutting it down they are planning to release it in China only with Tencent partnership. If Amazon has failed why not just have something like APKMirror to get the apps or leave the storefront to us. This is beyond stupid and another massive mistake after Lumia.

I think Tencent is not using WSA. They use Android 12 and Intel bridge. I need to turn off VM platform to use it. The APPs can not act like native Windows APPs.I can't use ADB to connect it.

**Comment by lisyclover** [comment]
> > @hifihedgehog well said! WSA could've been one of the core pillars of Windows especially with the new ARM chips. It's mind-blowing and emulators don't even come close. The fact that you cannot easily differentiate between a native and a WSA app says how good it was in terms of support and everything.
> > After shutting it down they are planning to release it in China only with Tencent partnership. If Amazon has failed why not just have something like APKMirror to get the apps or leave the storefront to us. This is beyond stupid and another massive mistake after Lumia.
> 
> I think Tencent is not using WSA. They use Android 12 and Intel bridge. I need to turn off VM platform to use it. The APPs can not act like native Windows APPs.I can't use ADB to connect it.

You are talking about the old version. The new version is waiting to be released in the Windows Store, and the kernel that should be used is wsa

**Comment by lyc280705** [comment]
> > > @hifihedgehog well said! WSA could've been one of the core pillars of Windows especially with the new ARM chips. It's mind-blowing and emulators don't even come close. The fact that you cannot easily differentiate between a native and a WSA app says how good it was in terms of support and everything.
> > > After shutting it down they are planning to release it in China only with Tencent partnership. If Amazon has failed why not just have something like APKMirror to get the apps or leave the storefront to us. This is beyond stupid and another massive mistake after Lumia.
> > 
> > I think Tencent is not using WSA. They use Android 12 and Intel bridge. I need to turn off VM platform to use it. The APPs can not act like native Windows APPs.I can't use ADB to connect it.
> 
> You are talking about the old version. The new version is waiting to be released in the Windows Store, and the kernel that should be used is wsa

Thank you for informing that.

**Comment by 1115714829** [comment]
事实上现在新的版本已经发布，Windows应用商店的应用宝 就是 安卓应用， 然而在兼容性检查中明确显示需要关闭hyper-v才支持使用。In fact, the new version has now been released. The Tencent App Treasure (应用宝) in the Windows Store ‌is‌ Android apps. However, the compatibility check clearly shows that Hyper-V needs to be disabled to enable their use.
